pkg/controller/certificates/certificate_controller.go

```diff
@@ -61,8 +62,12 @@ func NewCertificateController(
 
 	cc := &CertificateController{
 		kubeClient: kubeClient,
-		queue:      workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "certificate"),
-		handler:    handler,
+		queue: workqueue.NewNamedRateLimitingQueue(workqueue.NewMaxOfRateLimiter(
+			workqueue.NewItemExponentialFailureRateLimiter(1*time.Second, 1000*time.Second),
```

Comment:
1s seems too slow for real, transient errors. I'm not sure what the right number should be. But the nice thing about exponential backoff is the behavior of 5ms vs 1s for a non-transient error is not that different (other than the annoying logging).

Reply:
200ms seems like a reasonable choice.